ludwig/datasets/insurance_lite/__init__.py

```diff
@@ -47,26 +44,29 @@ def __init__(self, cache_dir=DEFAULT_CACHE_LOCATION):
 
     def process_downloaded_dataset(self):
         makedirs(self.processed_temp_path, exist_ok=True)
-        
-        url = self.config['s3_download_urls']
-        file_name = os.path.join(self.raw_dataset_path, url.split('/')[-1])
-        # TODO(shreya): DataFrame created twice: here + CSVMixin. Figure out
-        # options for using it once.
-        df = pd.read_csv(
-            file_name, header=0,
-            names=['id', 'image_path', 'insurance_company',
-                    'cost_of_vehicle', 'min_coverage', 'expiry_date',
-                    'max_coverage', 'condition', 'amount'],
-            usecols=list(range(1, 9))
+
+        for url in self.config['kaggle_dataset_files']:
+            file_name = os.path.join(self.raw_dataset_path, url.split('/')[-1])
+            # TODO(shreya): DataFrame created twice: here + CSVMixin. Figure out
+            # options for using it once.
+            df = pd.read_csv(
+                file_name, header=0,
+                names=['id', 'image_path', 'insurance_company',
+                        'cost_of_vehicle', 'min_coverage', 'expiry_date',
+                        'max_coverage', 'condition', 'amount'],
+                usecols=list(range(1, 9))
+                )
+            img_urls = df['image_path']
+            self.download_images(img_urls)
+            df['image_path'] = df['image_path'].apply(
+                lambda x: os.path.join(
+                    self.raw_dataset_path, 'images', x.split('/')[-1]
```

Comment:
Same here, cleaner to use `os.path.basename(x)`

Reply:
Done.